packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -214,10 +194,6 @@ const DomainPicker: FunctionComponent< Props > = ( {
 		}
 	}, [ allSuggestions, currentDomain ] );
 
-	useTrackModal( tracksName, onModalOpen, onModalUnmount, {
-		selected_domain: selectedDomain?.domain_name,
-	} );
```

Comment:
👍 
Since `DomainPicker` is not a modal (we only choose to wrap it inside a modal in Gutenboarding) getting this out feels right.